frontend/src/components/pages/VolunteerScheduling/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { Container, Text } from "@chakra-ui/react";
```

Comment:
i moved all of the files for the volunteer shift sign up flow in a separate folder (similar to how the donor `Dashboard` and `Scheduling` are separated). not sure if this creates too much conflict or is unneccesary

Reply:
I agree with that!